oauth2-code-flow-sample/login.html

```diff
@@ -1,3 +1,8 @@
+<!-- 
+  Copyright 2022 Volvo Car Corporation
+  SPDX-License-Identifier: Apache-2.0
+-->
+
 <!DOCTYPE html>
```

Comment:
Better that DOCTYPE is the very first tag in the html and comment after it. Some parsers can behave wonky if DOCTYPE isn't on top (most old ones, but you never know who's going to use what...).